package.json

```diff
@@ -5,24 +5,23 @@
     "private": true,
     "license": "BSD-3-Clause",
     "dependencies": {
-        "@dhis2/analytics": "4.3.25",
+        "@dhis2/analytics": "^4.4.1",
         "@dhis2/app-runtime": "^2.2.1",
         "@dhis2/d2-i18n": "^1.0.6",
-        "@dhis2/d2-ui-core": "^6.5.8",
-        "@dhis2/d2-ui-interpretations": "^6.5.7",
-        "@dhis2/d2-ui-mentions-wrapper": "^6.0.1",
-        "@dhis2/d2-ui-rich-text": "^6.0.1",
-        "@dhis2/d2-ui-sharing-dialog": "^6.0.1",
-        "@dhis2/d2-ui-translation-dialog": "^6.0.1",
-        "@dhis2/data-visualizer-plugin": "^34.3.34",
+        "@dhis2/d2-ui-core": "^7.0.3",
+        "@dhis2/d2-ui-interpretations": "^7.0.3",
+        "@dhis2/d2-ui-mentions-wrapper": "^7.0.3",
+        "@dhis2/d2-ui-rich-text": "^7.0.3",
+        "@dhis2/d2-ui-sharing-dialog": "^7.0.3",
+        "@dhis2/d2-ui-translation-dialog": "^7.0.3",
+        "@dhis2/data-visualizer-plugin": "^34.3.32-alpha.2",
```

Comment:
I see this is a downgrade - and also, how can I go back and check what's in this version?

Reply:
now upgraded to the correct version 34.5.2